aten/src/ATen/cuda/CUDAGraph.cpp

```diff
@@ -83,7 +83,7 @@ void CUDAGraph::capture_begin(MempoolId_t pool/*=0*/) {
   // cudaStreamCaptureModeGlobal is the most conservative option to
   // prevent potentially unsafe CUDA API calls during capture.  See
   // https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__STREAM.html#group__CUDART__STREAM_1g9d0535d93a214cbf126835257b16ba85
-  AT_CUDA_CHECK(cudaStreamBeginCapture(capture_stream_, cudaStreamCaptureModeGlobal));
+  AT_CUDA_CHECK(cudaStreamBeginCapture(capture_stream_, cudaStreamCaptureModeRelaxed));
```

Comment:
Is this meant to be done unconditionally, or just for ROCm?

Reply:
This was a test before the capture mode change was committed.  I'll move this back to cudaStreamCaptureModeGlobal